packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePicker12HourMobile.test.scss

```diff
@@ -0,0 +1,5 @@
+:local {
```

Comment:
We should update the test to set this color to transparent and remove the custom style binding for this example. This can be added to that test in the before hook:
```
before(() => {
        // Removes the blinking cursor to prevent screenshot mismatches.
        browser.execute('document.querySelector("input").style.caretColor = "transparent";');
});
```

Reply:
For reference: https://github.com/cerner/terra-core/blob/master/packages/terra-search-field/tests/wdio/search-field-spec.js#L12

Reply:
the before() method/hook is part of the wdio library which cannot be used in the dev-site component test files.
So it is not possible to implement the same in the above file.

Reply:
No we should implement this transparent style in the wdio test spec itself and remove the style completely from this example.

Reply:
Logged an issue for this: https://github.com/cerner/terra-framework/issues/739

Planning to merge this PR as-is, we can follow up with another PR to make sure we're consistent with how we style the caret in form fields during WDIO tests.